src/main/java/org/myrobotlab/service/abstracts/AbstractServo.java

```diff
@@ -601,7 +602,8 @@ public Double moveTo(Double newPos) {
       return newPos;
     }
 
-    boolean validMoveRequest = processMove(newPos, false, null);
+    broadcast("publishServoMoveTo", newPos);
```

Comment:
without the name of the servo publishing this, this is a pretty useless broadcast, no?

Reply:
Still useful to the UI which inherently knows which servo ... but I totally agree - it should return a tuple that has name and position ... (future pr)